picasso/src/main/java/com/squareup/picasso/ImageViewAction.java

```diff
@@ -44,6 +44,8 @@
       return;
     }
 
+    result.prepareToDraw();
```

Comment:
This seems pretty late to call this. Can we do it before posting to the main thread?